docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -34,9 +34,22 @@ Globally cached lookups can be specified as part of the [cluster wide config for
          "format": "csv",
          "columns": [
            "key",
-           "value"
+           "val1",
+           "val2"
          ]
        },
+       "maps":[
+         {
+           "mapName":"default",
+           "keyColumn":"key",
+           "valueColumn":"val1"
```

Comment:
that seems like a reasonable change, it would express more directly that simpleJson parser doesn't use the "maps" field unlike the other parser types

I suppose the logic for map building from a set of KeyValueMaps in the URIExtractionNamespace's delegate parser could be moved to something shared by the CSV/TSV/customJson "FlatDataParsers" in URIExtractionNamespace